ee/api/test/test_conversation.py

```diff
@@ -548,3 +585,52 @@ def test_get_throttles_skips_rate_limits_for_debug_mode(self):
         throttles = viewset.get_throttles()
         self.assertNotIsInstance(throttles[0], AIBurstRateThrottle)
         self.assertNotIsInstance(throttles[1], AISustainedRateThrottle)
+
+    def test_billing_context_validation_valid_data(self):
+        """Test that valid billing context data is accepted."""
+
+        with patch("ee.api.conversation.Assistant.astream", return_value=_async_generator()):
```

Comment:
Question: As you mock astream, it seems to bypass the integration flow through ConversationStreamManager/Temporal. Is it ok/tested in a different place, or would it be an overkill in general?